lib/KafkaBacklogMetrics.js

```diff
@@ -0,0 +1,467 @@
+const async = require('async');
+const { EventEmitter } = require('events');
+const zookeeper = require('node-zookeeper-client');
+
+const Logger = require('werelogs').Logger;
+const errors = require('arsenal').errors;
+
+const zookeeperHelper = require('./clients/zookeeper');
+
+const CheckConditionError = {};
+const NoNodeError = {};
```

Comment:
Why not make these instances of `Error`?

Reply:
I could, but I don't know if that would bring any value, but I'm open to changing this if it does! I though since those errors never leave the inner class functions, they can be simple empty objects.

Reply:
Definitely it's functionally the same. In my mind I was expecting the objects to be populated with values in some way, so by making them an `Error` I would know they'd be used as such. It's a minor readability point though.

Reply:
done